packages/store/src/store.ts

```diff
@@ -87,7 +98,9 @@ export class Store {
    * Allow the user to subscribe to the root of the state
    */
   subscribe(fn?: (value: any) => void): Subscription {
-    return this._stateStream.pipe(leaveNgxs(this._internalExecutionStrategy)).subscribe(fn);
+    return this._selectableStateStream
+      .pipe(leaveNgxs(this._internalExecutionStrategy))
```

Comment:
Is this still necessary since we are using the new shared stream?

Reply:
This is necessary in case the Store class is subscribed outside of the Angular zone.

Reply:
I thought that the new `_selectableStateStream` would handle ensuring that the subscribers are notified in the correct zone?
Or is this for the case where we are reading the current value that already exists in the stream?
If this is the case, then would we not also have to do this if the `store.select` is subscribed to outside of the Angular Zone?

Reply:
I thought that the new `_selectableStateStream` would handle ensuring that the subscribers are notified in the correct zone? But maybe there would be no guarantee for new subscriptions where we first receive the current value that already exists in the stream?
If this is the case, then would we not also have to do this if the `store.select` is subscribed to outside of the Angular Zone?

Reply:
The `_selectableStateStream` cannot ensure that completely. Given this example:
```ts
constructor(store: Store, ngZone: NgZone) {
  ngZone.runOutsideAngular(() => {
    setTimeout(() => {
      store.subscribe(() => console.log(Zone.current)); // will log angular
    }, 3000);
  });
}
```
Otherwise, it'll log `<root>`.